src/screens/FormField.js

```diff
@@ -286,6 +287,11 @@ const FormFieldPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="formField.checkBox.pad">
+          <Description>The CheckBox pad when wrapped in FormField.</Description>
+          <GenericPad />
+        </Property>
+
```

Comment:
Should we add this to the CheckBox doc so it is in both places?

Reply:
yeah we can I was back and forth on that